dev/source/docs/building-setup-windows10.rst

```diff
@@ -38,124 +39,139 @@ WSL Setup Steps
 
 #. Install an XWindows application to run graphical programs such as SITL by installing `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_, `Cygwin X <https://x.cygwin.com/>`_ or `Xming <https://sourceforge.net/projects/xming/>`_ on Windows.
 
-#. Open the "Ubuntu" application and add the following to the end of ``~/.bashrc``. Be sure to comment and uncomment the appropriate lines depending on if you are using WSL1 or WSL2 (if in doubt, WSL1 is the default)
+#. Open the "Ubuntu" application and add the following to the end of ``~/.bashrc``. Be sure to comment and uncomment the appropriate lines depending on if you are using WSL1 or WSL2.
 
-    .. code-block:: bash
+.. code-block:: bash
 
-        # Export Display for XWindows
-        # For WLS1
-        export DISPLAY=0:0
-        # For WSL2
-        # export DISPLAY=$(grep -m 1 nameserver /etc/resolv.conf | awk '{print $2}'):0
+    # Export Display for XWindows
+    # For WLS1
+    # export DISPLAY=0:0
+    # For WSL2
+    export DISPLAY=$(grep -m 1 nameserver /etc/resolv.conf | awk '{print $2}'):0
```

Comment:
Also since you are editing this can we default to WSL2 please :)

Reply:
I think that is the default in Windows now? I use 1 just because of restricted office computer stuff. I haven't tried Tom's board upload trick at home yet. Was there a specified version difference I missed between 1 and 2?

Reply:
WSL2 should be the default when you install it.  And WSL1 is only really needed if you want to upload from command line (easily).  But in general WSL2 is faster I believe.

Reply:
WSL2 uploads work per this PR https://github.com/ArduPilot/ardupilot/pull/19979